book/src/faqs/faqs.md

```diff
@@ -0,0 +1,40 @@
+# FAQs
+
+*Additional details will be published here and on our Medium as new products and tools launch.*
+
+---
+
+## Team
+
+### Where can I find more information about the Composable Team?
+
+If you’ve already checked out “[The Composable Team](../ecosystem/the-composable-team.html)” tab, you can find more information using the resources below:
+
+- Advanced Blockchain AG’s homepage is [here](https://www.advancedblockchain.com/), with their complete portfolio [here](https://www.advancedblockchain.com/portfolio), and their newsroom of announcements and updates [here](https://blog.advancedblockchain.com/).
+- The experiences of our Co-Founder and Lead Developer Cosmin can be found on his [LinkedIn](https://www.linkedin.com/in/grigorecosmin?originalSubdomain=ro).
+
+Our [Head of Product](https://twitter.com/0xbrainjar) and [Lead Polkadot Developer](https://twitter.com/0Xvatbrain) can be followed on twitter.
+
+---
+
+## Company Information
+
+### What is Composable Finance?
+
+Composable Finance is a cross-chain, cross-layer, comprehensive DeFi infrastructure, powered by Layer 2 Ethereum and Polkadot. Our ultimate goal is to make the seamlessly interoperable future of DeFi into a reality, via our suite of product offering intended to resolve particular pain points in interoperability.
+
+### How is Composable funded?
+
+The most recent round of funding for Composable Finance raised a total of $7 million. This is an incredible step for the company, pushing it into the future and towards new advancements. 
+
+Leading this round were [Advanced Blockchain AG](https://en.advancedblockchain.com/) and [Rarestone Capital](https://rarestone.capital/), both of which are key venture capitalists investing in the blockchain industry. Together these organizations used strategic investors to make this funding round a success.
+
+### How was Composable Finance created?
+
+Through its subsidiaries, [Advanced Blockchain AG](https://en.advancedblockchain.com/) incubated Composable Finance on the technical side and provided the intellectual property for Composable. During this period, Cosmin Grigore was the Lead Developer on the project. This incubation was completed alongside top-tier Web 3.0-focused venture capital firm [Rarestone Capital](https://rarestone.capital/), which assisted with product strategy and rollout for Composable Finance. Subsequently, Composable Finance was spun out, retaining Cosmin as its CEO. Advanced Blockchain also reinvested into Composable. 
+
+Are there any socials I can follow for any additional questions and get more updates and information?
```

Comment:
should this be a header or something? maybe a quote (`>`)

Reply:
done